static/js/components/modal-gallery.js

```diff
@@ -0,0 +1,29 @@
+import $ from 'jquery';
```

Comment:
Instead of porting over the custom JS we'd written, which was pretty heavy with statemachine stuff, I just instead pulled in a simple / small Lightbox plugin.